src/common.rs

```diff
@@ -124,6 +124,7 @@ pub struct SignatureShare {
     pub key_ids: Vec<u32>,
 }
 
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
```

Comment:
Are you actually planning on naively using `Serialize` and `Deserialize`?  That will not end well for you.

Reply:
Can you explain? These traits are derived for both `Point`, `Scalar`, and most of the other structs in this crate. Is there something different about `Signature` that I'm missing?

Reply:
All of the current `Deserialize` and `Serialize` implementations are not particularly well implemented.  Basically, the internal representations are moderately platform dependent, but they should be okay for any currently used platform, so a naive `serde` derive *should* be okay.  So I've neglected actually fixing this.

I have a PR which I'm about to merge which does fix this:

https://github.com/Trust-Machines/p256k1/pull/75

Once that's in I'll update this repo to also do the correct `serde` magic.

Reply:
https://github.com/Trust-Machines/wsts/pull/38